src/devices/Card/Ndef/MessageFlag.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ndef
+{
+    /// <summary>
+    /// Message flag
+    /// </summary>
+    [Flags]
+    public enum MessageFlag
```

Comment:
should this be internal?

Reply:
No if you want to adjust a message yourself. Keep in mind NDEF can be used on other cards than Mifare (even if I never saw that). So you may want to create your own messages and play with them independently of the embedded logic. Like changing the order of the messages.